src/main/java/seedu/eylah/expensesplitter/model/receipt/Receipt.java

```diff
@@ -48,6 +56,19 @@ public Entry getEntry(int index) {
         return this.receipt;
     }
 
+    /**
+     * Returns true if both persons of the same name have at least one other identity field that is the same.
+     * This defines a weaker notion of equality between two persons.
+     */
+    public boolean isSameReceipt(Receipt otherReceipt) {
```

Comment:
probably as of now we dont need isSameReceipt if we are only having 1 receipt right? Maybe we can extend to multiple receipt but not in v1.3 as we only have 2 more days to deadline and we still have a lot try catch / user input exception handling to settle. 

But even if we do extend to include multiple receipts: 2 receipts each having Pasta split among willy shi and ailan, same price will be considered same but it should not be "detected as duplicate and hence deleted/unable to create" because we can have similar receipt with similar Entries right?

Reply:
This is a skeleton method used by uniqueReceiptList. Currently, this method haven't fully implemented as if we want to do the multiple receipts, we need the name for each receipt to identify them.